packages/docs/components/atoms/LargeText/index.md

```diff
@@ -1,13 +1,24 @@
-# LargeText (todo) <Badges :texts="badges" />
+# LargeText <Badges :texts="badges" />
 
 <script setup>
   import pkg from '@studiometa/ui/atoms/LargeText/package.json';
   const badges = [`v${pkg.version}`, 'JS', 'Twig'];
 </script>
 
+Use the large text component to easily add very nice looking title animation.
+
 ## Table of content
 
 - [Examples](./examples.md)
 - [JS API](./js-api.md)
 - [Twig API](./twig-api.md)
```

Comment:
**unrelated to this PR/question** Is this way of structuring pages is the most helpful ? 
I'm wondering if it's good to have user going back and forth between multiple pages to view APi's and exemples. 
Any opinion on the subject @studiometa/core ?

Reply:
We can add examples on the API pages, the examples pages can be a way to show more advanced or complex usages.

Reply:
Can we do that in the next documentation PR ?

Reply:
sure, this is not blocking